next.config.js

```
@@ -111,10 +116,10 @@ const nextConfig = {
     },
 }
 
-const { withSentryConfig } = require('@sentry/nextjs')
+if (process.env.NODE_ENV !== 'development' && !Boolean(process.env.LOCAL_BUILD)) {
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)